documentation/api/query/examples-pql.markdown

```diff
@@ -0,0 +1,447 @@
+        "certname": "greenserver.vm",
+        "deactivated": null,
+        "expired": null,
+        "facts_environment": "production",
+        "facts_timestamp": "2016-08-15T11:06:26.140Z",
+        "latest_report_corrective_change": null,
+        "latest_report_hash": "4a956674b016d95a7b77c99513ba26e4a744f8d1",
+        "latest_report_noop": false,
+        "latest_report_noop_pending": null,
+        "latest_report_status": "changed",
+        "report_environment": "production",
+        "report_timestamp": "2016-08-15T11:06:18.393Z"
+    }
+]
+```
+
+***
+
+### Mcollective sub-collective filtering
+
+Show nodes that belong in the sub-collective `dc1`.
+
+``` ruby
+inventory[certname] { facts.mcollective.server.collectives[*] = "dc1" } }
+```
+
+*Output:*
+
+``` json
+[
+    {
+        "certname": "greenserver.vm"
+    },
+    {
+        "certname": "yellowserver.vm"
+    }
+]
+```
+
+***
+
+### Profile querying
+
+Show nodes that have the profile class `Profile::Remote_mgmt` applied to it.
+
+``` ruby
+nodes { resources { type = "Class" and title = "Profile::Remote_mgmt" } }
+```
+
+*Output:*
+
+``` json
+[
+    {
+        "cached_catalog_status": "not_used",
+        "catalog_environment": "production",
+        "catalog_timestamp": "2016-08-15T11:06:26.275Z",
+        "certname": "greenserver.vm",
+        "deactivated": null,
+        "expired": null,
+        "facts_environment": "production",
+        "facts_timestamp": "2016-08-15T11:06:26.140Z",
+        "latest_report_corrective_change": null,
+        "latest_report_hash": "4a956674b016d95a7b77c99513ba26e4a744f8d1",
+        "latest_report_noop": false,
+        "latest_report_noop_pending": null,
+        "latest_report_status": "changed",
+        "report_environment": "production",
+        "report_timestamp": "2016-08-15T11:06:18.393Z"
+    }
+]
+```
+
+***
+
+### Querying catalog submission time
+
+Check for nodes that haven't had a catalog applied since a certain time.
+
+``` ruby
+nodes { catalog_timestamp < "2016-08-15T11:37:00.000Z" }
+```
+
+*Output:*
+
+``` json
+[
+    {
+        "cached_catalog_status": "not_used",
+        "catalog_environment": "production",
+        "catalog_timestamp": "2016-08-15T11:06:26.275Z",
+        "certname": "greenserver.vm",
+        "deactivated": null,
+        "expired": null,
+        "facts_environment": "production",
+        "facts_timestamp": "2016-08-15T11:06:26.140Z",
+        "latest_report_corrective_change": null,
+        "latest_report_hash": "4a956674b016d95a7b77c99513ba26e4a744f8d1",
+        "latest_report_noop": false,
+        "latest_report_noop_pending": null,
+        "latest_report_status": "changed",
+        "report_environment": "production",
+        "report_timestamp": "2016-08-15T11:06:18.393Z"
+    }
+]
+```
+
+***
+
+### List of nodes with report status failed
+
+List all nodes that have had a failure on their last run.
+
+``` ruby
+nodes { latest_report_status = 'failed' }
+```
+
+*Output:*
```

Comment:
But those examples you have listed are in the same box because they are a CLI command and a resulting response. An example query in this page isn't the same. The examples aren't CLI commands basically.